packages/client-twitter/src/post.ts

```diff
@@ -11,25 +11,24 @@ import {
 import { elizaLogger } from "@ai16z/eliza";
 import { ClientBase } from "./base.ts";
 
-const twitterPostTemplate = `{{timeline}}
-
-# Knowledge
+const twitterPostTemplate = `
+# Areas of Expertise
 {{knowledge}}
 
-About {{agentName}} (@{{twitterUserName}}):
+# About {{agentName}} (@{{twitterUserName}}):
 {{bio}}
 {{lore}}
-{{postDirections}}
+{{topics}}
 
 {{providers}}
 
-{{recentPosts}}
-
 {{characterPostExamples}}
 
-# Task: Generate a post in the voice and style of {{agentName}}, aka @{{twitterUserName}}
-Write a single sentence post that is {{adjective}} about {{topic}} (without mentioning {{topic}} directly), from the perspective of {{agentName}}. Try to write something totally different than previous posts. Do not add commentary or acknowledge this request, just write the post.
-Your response should not contain any questions. Brief, concise statements only. No emojis. Use \\n\\n (double spaces) between statements.`;
+{{postDirections}}
+
+# Task: Generate a post in the voice and style and perspective of {{agentName}} @{{twitterUserName}}.
+Write a 1-3 sentence post that is {{adjective}} about {{topic}} (without mentioning {{topic}} directly), from the perspective of {{agentName}}. Do not add commentary or acknowledge this request, just write the post.
+Your response should not contain any questions. Brief, concise statements only. The total character count MUST be less than 280. No emojis. Use \\n\\n (double spaces) between statements.`;
```

Comment:
a bit cleaner would be something like this. Maybe we can fine-tune or A/B test the prompt. 

`Write a concise, engaging post that is {{adjective}} about {{topic}} without mentioning {{topic}} directly. The post should be 1-3 sentences long, maintaining a coherent and authentic tone that reflects {{agentName}}'s personality and expertise. Avoid adding any commentary or acknowledgment of this request. Ensure the total character count is less than 280 characters. Use \\n\\n (double spaces) between statements. No emojis or questions allowed.`

Reply:
I can try this out